Remove build flags that are redundant and disable optimization flags when debugging is enabled.

From: Roman Donchenko <roman.donchenko@itseez.com>


---
 Platform/Android/jni/Application.mk |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Platform/Android/jni/Application.mk b/Platform/Android/jni/Application.mk
index 6d2da24..91e2646 100644
--- a/Platform/Android/jni/Application.mk
+++ b/Platform/Android/jni/Application.mk
@@ -5,12 +5,13 @@ APP_PLATFORM := android-9
 
 # Build ARMv7-A machine code.
 APP_ABI := armeabi-v7a
-APP_CFLAGS := -O3 -ftree-vectorize -ffast-math -funroll-loops
 
-APP_CFLAGS += -fPIC
+ifndef NDK_DEBUG
+APP_CFLAGS := -O3 -ftree-vectorize -ffast-math -funroll-loops
+endif
 
 ifeq ($(APP_ABI),armeabi-v7a)
-	APP_CFLAGS += -march=armv7-a -mfloat-abi=softfp -mtune=cortex-a9 -mfp=vfpv3-d16 -mfpu=vfp
+	APP_CFLAGS += -mtune=cortex-a9
 
 # optionally add NEON to compilation flags.
 # to activate, run: "ndk-build USE_NEON=1"
